Philips.CodeAnalysis.MaintainabilityAnalyzers/Readability/LimitConditionComplexityAnalyzer.cs

```diff
@@ -22,7 +22,7 @@ public class LimitConditionComplexityAnalyzer : DiagnosticAnalyzer
 	{
 		private const string Title = "Limit the number of clauses in a condition";
 		private const string Message =
-			"Limit the number of clauses in the condition statement, to not more than {0}.";
+			"Found {0} clauses in the condition statement. Limit the number of clauses in the condition statement, to not more than {1} (or define max_operators in .editorconfig to customize).";
```

Comment:
If you configured a maximum via .editorconfig, you shouldn't get this suggestion. I think it is fair to remove the parenthesized text here. Maybe add a reference to the .editorconfig items for this Analyzer in the class diagram ? Not sure how other Analyzers steer their users to the .editorconfig options they provide.